modules/installation-upi-machines-iso.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-upi-machines-iso-{context}"]
+= Creating {op-system-first} machines using an ISO image
+
+Before you install a user provisioned infrastructure (UPI) cluster on bare
+metal, you need to create {op-system} machines for it to use. You can use an
+ISO image to create the machines.
+
+.Prerequisites
+
+* Obtain the Ignition config files for your cluster.
+* Have access to an HTTP server that you can access from your computer and that
+the machines that you create can access.
+
+.Procedure
+
+////
+This will work for GA.
+. Obtain the {op-system} images from the
+link:https://access.redhat.com/downloads[Product Downloads] page on the Red
+Hat customer portal.
+////
+
+. Upload the control plane, compute, and bootstrap Ignition config files that the
+installation program created to your HTTP server. Note the URLs of these files.
+
+. Obtain the {op-system} ISO image:
++
+----
+$ curl -J -L -O https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190412.1/rhcos-410.8.20190412.1-installer.iso
+----
+
+. Use the ISO to install to start the {op-system} installation. Use one of the following
+installation options:
+** Burn the ISO image to a disk and boot it directly.
+** Use ISO redirection via a LOM interface.
+
+. After the instance boots, press the `TAB` or `E` key to edit the kernel command line.
+. Add the parameters to the kernel command line:
++
+----
+coreos.inst=yes
+coreos.inst.install_dev=sda <1>
+coreos.inst.image_url=https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190412.1/rhcos-410.8.20190412.1-<ext> <2>
+coreos.inst.ignition_url=http://example.com/config.ign <3>
+----
+<1> Specify the block device of the system to install to.
+<2> Specify the URL of the image. Replace `<ext>` with the extension that
+corresponds to your required file type, either `metal-bios.raw.gz` or `metal-uefi.raw`
+<3> Specify the URL of the Ignition config file for this machine type.
+
+. Press Enter to complete the installation. After {op-system} installs, the system
```

Comment:
`Enter`

Reply:
@ahardin-rh, is this one of our local overrides? IBM Style says:
> Do not highlight the names of keys.

Reply:
@kalexand-rh No, I think you're correct on this one. IBM Style wins. Thanks!

Reply:
Thank you!